app/Models/Configuration.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Models;
+
+use App\Enums\ConfigValueType;
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Configuration extends Model
```

Comment:
```suggestion
class UserConfiguration extends Model
```


Minor change request in case we ever want to implement other types of admin controlled configs.